ui/packages/catalog-ui-search/src/main/webapp/component/input/geometry/input-geometry.less

```diff
@@ -1,18 +1,22 @@
 @{customElementNamespace}input.is-geometry {
-    .error-form,
-    .error-ordering {
-        display: none;
-    }
+  label {
+    padding-left: 10px;
```

Comment:
❓ should we be using `em` instead of `px` so these scale better with zoom/density?